src/Gateway/CustomerGateway.php

```diff
@@ -0,0 +1,28 @@
+<?php
+declare(strict_types=1);
+
+namespace Twikey\Api\Gateway;
+
+use Psr\Http\Client\ClientExceptionInterface;
+use Twikey\Api\Exception\TwikeyException;
+
+class CustomerGateway extends BaseGateway
+{
+    /**
+     * @throws TwikeyException
+     * @throws ClientExceptionInterface
+     */
+    public function update($data)
+    {
+        if (!isset($data['customerNumber'])) {
+            throw new TwikeyException("A customerNumber is required");
+        }
+        $response = $this->request(
+            'PATCH',
+            \sprintf("creditor/customer/%s", $data['customerNumber']),
```

Comment:
Is it possible that the path lacks a / in the beginning?

Reply:
Hi Bonno, just corrected the base function as indeed there was a slash to much. Could you add the slash in the beginning this way I can merge your pr.

Reply:
@koen-serry added the slash at the beginning of the path